roles/reproducer/tasks/ci_deploy_data.yml

```diff
@@ -29,7 +29,7 @@
       }}
   ansible.builtin.set_fact:
     cacheable: true
-    ci_job_networking: "{{ zuul_inventory.all.hosts.crc.crc_ci_bootstrap_networking }}"
+    ci_job_networking: "{{ zuul_inventory.all.hosts.controller.crc_ci_bootstrap_networking }}"
```

Comment:
Note this is a quick workaround - I'll try to find a better suited solution. There are DS jobs running and zuul inventory doesn't seem to have crc node, just "controller", leading to a crash in case of a reproducer run.

I'm waiting to get some feedback about that workaround, in order to implement a better solution.